### PR TITLE
Capture logs in integration tests

### PR DESCRIPTION
**Pull request by RReverser**
# Description of Changes

This helps debug failures.

# API and ABI

 - [ ] This is a breaking change to the module ABI
 - [ ] This is a breaking change to the module API
 - [ ] This is a breaking change to the ClientAPI
 - [ ] This is a breaking change to the SDK API

*If the API is breaking, please state below what will break*

# Expected complexity level and risk

1

*How complicated do you think these changes are? Grade on a scale from 1 to 5,
where 1 is a trivial change, and 5 is a deep-reaching and complex change.*

*This complexity rating applies not only to the complexity apparent in the diff,
but also to its interactions with existing and future code.*

*If you answered more than a 2, explain what is complex about the PR,
and what other components it interacts with in potentially concerning ways.*

### CODE REVIEW

File: crates/testing/tests/standalone_integration_test.rs [hunk 1]
```diff
@@ -2,10 +2,21 @@ use serde_json::Value;
 use serial_test::serial;
 use spacetimedb_testing::modules::{CompilationMode, CompiledModule, DEFAULT_CONFIG};
 
+fn init() {
+    let _ = env_logger::builder()
+        .parse_filters(
+            "spacetimedb=trace,spacetimedb_client_api=trace,spacetimedb_lib=trace,spacetimedb_standalone=trace",
+        )
+        .is_test(true)
+        .try_init();
```
Comment (gefjon) [inline crates/testing/tests/standalone_integration_test.rs:11]:
```suggestion
        // `try_init` and ignore failures to continue if a logger is already registered.
        // This allows us to call `init` at the start of every test without a `once_cell` or similar.
        .try_init();
```

**Review by gefjon — COMMENTED** [review]
Complete the PR template and I'll approve.

**Comment by RReverser** [comment]
> Complete the PR template and I'll approve.

I wonder - why did we add that field? It seems kinda useless in majority of cases as most PRs are simple changes.

**Comment by gefjon** [comment]
> I wonder - why did we add that field? It seems kinda useless in majority of cases as most PRs are simple changes (and if they're not, they should be split up into smaller ones anyway).

Maybe you weren't here for that meeting, but we added it in the hopes of making review easier by encouraging people to think about the complexity of their PRs. We had an endemic problem with people creating huge PRs that were difficult to review because they touched many parts of the codebase. That made reviewing more difficult even for the easy PRs, because it was hard to tell what you were getting into when you sat down to do a review.

We also had problems with PRs that appeared simple, but actually had non-obvious interactions with parts of the codebase that hadn't been directly edited. The PR authors were generally aware of those interactions, but the reviewers often were not, so we got some bugs that weren't caught by reviewers who didn't know where to look.

I pushed for the new template field because I figure on easy PRs it takes a few seconds to type a 1 or 2 in there, and on more complex PRs it's valuable. Adoption has been slow, so I'm trying to bully people into building the habit by refusing to approve any PRs that don't have it filled in, even for trivial PRs like this one.

**Comment by RReverser** [comment]
(Anyway, I did finish template so just waiting for the review.)

**Pull request merged**